### DISCUSSION TITLE
More shortcut in-game and UX Improvement

### DISCUSSION THREAD

**Discussion by fizray** [ppy/osu#21630] [Ideas] [answered]
I'd like to suggest making all in-game shortcut can be customized. And i see the Lazer mod menu selection doesn't have shortcut to reset all mod using "1" like the stable have, it really bothered me a lot though when changing a mod. And I'd like to see the personal preset section have keyboard shortcut to make preset selection faster, it's cool to have though.

Lazer mod selection (No number shortcut)
![osu_2022-12-13_21-20-27](https://user-images.githubusercontent.com/63831260/207358025-5286192c-41d3-4193-9882-cd7fe5737f1f.jpg)
Stable mod selection (Have number shortcut)
![screenshot362](https://user-images.githubusercontent.com/63831260/207358097-1159d101-2f6d-4546-acf4-e489871b8fa9.jpg)

**Comment by peppy** [comment, marked as answer]
Personal presets would work well with numbers, if we are willing to drop the "1 to reset" legacy binding.
